nerfstudio/data/utils/dataloaders.py

```diff
@@ -61,7 +63,15 @@ def __init__(
 
         self.cached_collated_batch = None
         if self.cache_all_images:
+            CONSOLE.print(f"Caching all {len(self.dataset)} images from the dataset.")
+            if len(self.dataset) > 500:
+                CONSOLE.print("Warning: If you run out of memory, try reducing the number of images to sample from.")
```

Comment:
Can you add "[bold yellow]Warning: If..."

Reply:
Sure! Maybe should learn more about this console... 😅

Reply:
It's a great library - https://github.com/Textualize/rich